basis/ui/gadgets/theme/theme.factor

```diff
@@ -6,4 +6,91 @@ IN: ui.gadgets.theme
 : theme-image ( name -- image-name )
     "vocab:ui/gadgets/theme/" prepend-path ".tiff" append <image-name> ;
 
-<< "ui.gadgets.theme.light" require >>
+SYMBOL: theme
+TUPLE: light ;
```

Comment:
If you're not going to have slots or inheritance, you can just make this a ``SINGLETON: light-theme``.

``light`` and ``dark`` are maybe ambiguous, so name them ``light-theme`` and ``dark-theme``?

Also, maybe define this in the light-theme file?